inc/functions.php

```diff
@@ -715,4 +715,27 @@ function colormag_site_icon_migrate() {
 }
 
 add_action( 'after_setup_theme', 'colormag_site_icon_migrate' );
+
+/**
+ * Migrate any existing theme CSS codes added in Customize Options to the core option added in WordPress 4.7
+ */
+function colormag_customm_css_migrate() {
+	if ( get_option( 'colormag_custom_css_transfer' ) ) {
+		return;
+	}
+
+	$theme_custom_css = get_theme_mod( 'colormag_custom_css', '' );
+	if ( ! empty( $theme_custom_css ) && function_exists( 'wp_update_custom_css_post' ) ) {
+		$wordpress_core_css = wp_get_custom_css(); // Preserve any CSS already added to the core option.
+		$return = wp_update_custom_css_post( $wordpress_core_css . $theme_custom_css );
+		if ( ! is_wp_error( $return ) ) {
+			// Set the transfer as complete
+			update_option( 'colormag_custom_css_transfer', 1 );
```

Comment:
Why we need transfer/migrate check. If custom css theme mod is empty then snippet is never going to execute again :)